tools/checked-c-convert/ProgramInfo.h

```diff
@@ -120,15 +180,16 @@ class ProgramInfo {
 
   // For each pointer type in the declaration of D, add a variable to the 
   // constraint system for that pointer type. 
-  bool addVariable(clang::Decl *D, clang::DeclStmt *St, clang::ASTContext *C);
+  bool addVariable(clang::DeclaratorDecl *D, clang::DeclStmt *St, clang::ASTContext *C);
 
   bool getDeclStmtForDecl(clang::Decl *D, clang::DeclStmt *&St);
 
   // Checks the structural type equality of two constraint variables. This is 
```

Comment:
Right now, nothing, since `checkStructuralEquality` always returns `false`. When it starts returning something other than `false`, it will probably have to perform the comparison on the LUB of `V` and `U`.